new/PRS14469TanseaKrestos.xml

```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14469TanseaKrestos" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Tanśǝʾa Krǝstos</title>
```

Comment:
The word for "Resurrection" is sometimes misspeled, but it happens rarely, and even more in the scribe's name. Perhaps this is a transcribing mistake of the cataloguer or a misprint, worth of a short comment.

Reply:
There are a lot of names with ተንሥአ፡ in the app.

Reply:
Yes, I checked. At least some of them originate from Transkribus which may misread ት of some scribal hands.